network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -86,7 +86,7 @@
 
                         {% for cat in categories %}
                             {% with selected_classes="active tw-bg-gray-80 tw-text-white tw-border-gray-80" default_classes="hover:tw-border-blue-10 hover:tw-bg-blue-10 tw-text-gray-60 tw-border-gray-20 tw-bg-white" tailwind_classes="tw-no-underline border tw-px-4 tw-py-2 tw-font-sans tw-rounded-3xl tw-font-normal tw-text-[12px] tw-leading-[1.3] tw-whitespace-nowrap" %}
-                                {% if not cat.parent %}
+                                {% if cat.parent != None %}
```

Comment:
I had to revert this back to what it was previously. It was not working and I don't why :man_shrugging: :house: :card_index: